cachelib/allocator/Refcount.h

```diff
@@ -247,10 +247,10 @@ class FOLLY_PACK_ATTR RefcountWithFlags {
   /**
    * The following four functions are used to track whether or not
    * an item is currently in the process of being moved. This happens during a
-   * slab rebalance or resize operation.
+   * slab rebalance or resize operation or during eviction.
    *
-   * An item can only be marked moving when `isInMMContainer` returns true.
-   * This operation is atomic.
+   * An item can only be marked moving when `isInMMContainer` returns true and
+   * the item is not yet marked as moving. This operation is atomic.
```

Comment:
hah, this is clever!

I will go through the code in more detail to reason about the logic change, but in theory this makes sense.

Minor comment. Please rename this API to `markExclusive`, since that is our intent now: marking an item as exclusive so cachelib can gain exclusive access to it internally.

Reply:
Hm, there is already a function `isExclusive` in Refcount.h Should we rename it to something else?

Reply:
I think we can deprecate `isExclusive()` altogether. This change actually calls for a much bigger refactor in the slab release code path.

Moving bit is now denoting "exclusive" access to an item. Slab release should also be using it for gaining exclusive access, and the current notion of holding an active refcount to prevent someone else from removing/evicting an item can be fully deprecated.

Will elaborate in separate places about slab release.